### PR TITLE
fix personalisation bug

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

**Review by ivov — APPROVED** [review]
Exactly, thank you.

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 1]
```diff
@@ -704,7 +704,7 @@
 		"thanks": "Thanks!",
 		"theseQuestionsHelpUs": "These questions help us tailor n8n to you",
 		"whichIndustriesIsYourCompanyIn": "Which industries is your company in?",
-		"whichOfTheseAreasDoYouMainlyWorkIn": "Which of these areas do you mainly work in?"
+		"whichOfTheseAreasDoYouMainlyWorkIn": "Which areas do you mainly work in?"
```
Comment (ivov) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:707]: Can we please have the key reflect the updated value?

**Review by ivov — COMMENTED** [review]
Just one little thing, otherwise LGTM. Thanks for the fixes.

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.158.0`

**Pull request merged**